readme.md

```diff
@@ -52,9 +52,51 @@ This project provides you with the scripts needed to deploy the necessary resour
 ````
 Speech commands have been published.
 ````
+You should see a SpeechApplicationId and a SpeechSubscriptionKey below the above message. Copy these down! You will need them for testing in the UWP app in the next section.
 
+# Testing with UWP App
+Once you have all the above resources created and deployed you are able to test using a UWP app. 
 
-## Testing
+## Things you need first 
+1. [Visual Studio](https://visualstudio.microsoft.com/vs/)
+2. Clone the [SpeechCommandAccelerator](https://github.com/negativeeddy/SpeechCommandAccelerator) repo from GitHub
+3. The SpeechApplicationId
+4. The SpeechSubscriptionKey
+
+## Setting up the UWP App
+
+1. In Visual Studio, open the Microsoft.CognitiveServices.Inventory.sln solution file. 
+
+2. In the "Solution Explorer", right click Microsoft.CognitiveServices.Inventory.UWP and select "Set as StartUp Project". Your "Solution Explorer" should look like this:
+![Visual Studio](doc/SEStartUp.png)
+
+3. Expand the Microsoft.CognitiveServices.Inventory project and open "appSettings.json". Update this file with your SpeechApplicationId and SpeechSubscriptionKey from above.
+
+4. Save 
+
+5. **Ensure Windows is in Develeper Mode** To build and deploy UWP apps locally, Windows needs to be in developer mode. You can ensure you have Developer Mode enabled by doing the following:
```

Comment:
i changed the developer mode to its own workflow so it can be skipped if they already know they have it.